README.md

```diff
@@ -303,36 +303,36 @@ If there currently isn't an active session, then calling this method does nothin
 Common Features

```

Comment:
Way at the top of this file is a table of contents that you'll need to update. User tracking needs to be changed to Customers (both what's displayed, and the anchor to link down to this section which doesn't currently work due to the name change down here).

Reply:
Good catch, changes this

Reply:
I would suggest also changing "User Tracking" to "Customers" to match the heading of that section.

Reply:
Ah yes, have changed this.